src/tbp/monty/frameworks/environments/embodied_data.py

```diff
@@ -576,6 +591,51 @@ def get_good_view(
         TODO M : move most of this to the motor systems, shouldn't be in embodied_data
             class
         """
+        if self._use_get_good_view_positioning_procedure:
+            _configured_policy = self.motor_system._policy
+
+            self.motor_system._policy = GetGoodView(
+                agent_id=self.motor_system._policy.agent_id,
+                desired_object_distance=_configured_policy.desired_object_distance,
+                good_view_percentage=_configured_policy.good_view_percentage,
+                multiple_objects_present=self.num_distractors > 0,
+                sensor_id=sensor_id,
+                target_semantic_id=self.primary_target["semantic_id"],
+                allow_translation=allow_translation,
+                max_orientation_attempts=max_orientation_attempts,
+                # TODO: Remaining arguments are unused but required by BasePolicy.
+                #       These will be removed when PositioningProcedure is split from
+                #       BasePolicy
+                #
+                # Note that if we use rng=self.rng below, then the following test will
+                # fail:
+                #   tests/unit/evidence_lm_test.py::EvidenceLMTest::test_two_lm_heterarchy_experiment  # noqa: E501
```

Comment:
We could just leave out the file directory parts `tests/unit` or even the whole file path and just reference the class and test method, and then we wouldn't need the `noqa` comment.